model/src/main/kotlin/licenses/LicenseClassifications.kt

```diff
@@ -56,6 +56,10 @@ data class LicenseClassifications(
                     getOrPut(category) { mutableSetOf() } += license.id
                 }
             }
+
+            categories.forEach { category ->
+                putIfAbsent(category.name, mutableSetOf())
+            }
```

Comment:
Just double-checking that this is no oversight as you did not comment on my proposed code in any way: It's intended that you did not change this, correct? I'm really just asking, without implying that I'd prefer that you should have changed it.

Reply:
Yes, I intentionally haven't changed that.